fields.go

```diff
@@ -41,7 +62,8 @@ func (fs *structFields) lookupByFoldedName(name []byte) []*structField {
 
 type structField struct {
 	id      int   // unique numeric ID in breadth-first ordering
-	index   []int // index into a struct according to reflect.Type.FieldByIndex
+	index0  int   // 0th index into a struct according to reflect.Type.FieldByIndex
+	index   []int // 1st index and remaining into a struct according to reflect.Type.FieldByIndex
```

Comment:
linkify the FieldByIndex references? I know that pkg.go.dev doesn't support linking to fields or methods, but gopls and other tools do support them, and they're still useful IMHO

Reply:
It seems `gopls` doesn't link to interface methods, which is a bug IMO.